Skype/WebSDK/docs/PTLocalUserLocation.md

```diff
@@ -1,4 +1,7 @@
-
+---
+title: Location
```

Comment:
I'd like there to be more than one word in the title, even though these articles are super old/legacy articles. So, maybe you can expand this to a few words?